benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1104,25 +1147,28 @@ void GraphicsBenchmarkApp::UpdateGUI()
 
 void GraphicsBenchmarkApp::DrawExtraInfo()
 {
-    uint64_t frequency = 0;
-    GetGraphicsQueue()->GetTimestampFrequency(&frequency);
+    if (HasActiveMetricsRun()) {
+        const auto cpuSubmissionTime = GetGaugeBasicStatistics(mMetricsData.metrics[MetricsData::kTypeCPUSubmissionTime]);
 
-    ImGui::Columns(2);
-    ImGui::Text("CPU Average Submission Time");
-    ImGui::NextColumn();
-    ImGui::Text("%.2f ms ", mSubmissionTime.average);
-    ImGui::NextColumn();
+        ImGui::Columns(2);
+        ImGui::Text("CPU Average Submission Time");
+        ImGui::NextColumn();
+        ImGui::Text("%.2f ms", cpuSubmissionTime.average);
+        ImGui::NextColumn();
 
-    ImGui::Text("CPU Min Submission Time");
-    ImGui::NextColumn();
-    ImGui::Text("%.2f ms ", mSubmissionTime.min);
-    ImGui::NextColumn();
+        ImGui::Text("CPU Min Submission Time");
+        ImGui::NextColumn();
+        ImGui::Text("%.2f ms", cpuSubmissionTime.min);
+        ImGui::NextColumn();
 
-    ImGui::Text("CPU Max Submission Time");
-    ImGui::NextColumn();
-    ImGui::Text("%.2f ms ", mSubmissionTime.max);
-    ImGui::NextColumn();
+        ImGui::Text("CPU Max Submission Time");
+        ImGui::NextColumn();
+        ImGui::Text("%.2f ms", cpuSubmissionTime.max);
+        ImGui::NextColumn();
+    }
 
+    uint64_t frequency = 0;
+    PPX_CHECKED_CALL(GetGraphicsQueue()->GetTimestampFrequency(&frequency));
     ImGui::Columns(2);
```

Comment:
nit: maybe this line and line 1153 could just be moved to right above the `HasActiveMetricsRun` if block.

Reply:
fixed